### DISCUSSION TITLE
Bug Report for Whisper Model - Chinese Transcription Anomaly

### DISCUSSION THREAD

**Discussion by xqiu** [openai/whisper#2244] [General]
Hello OpenAI Team,

I would like to report an anomaly I encountered with the Whisper model (v2/v3) while transcribing Chinese voice. The transcriptions often begin with the phrase "请不吝点赞 订阅 转发 打赏支持明镜与点点栏目", which seems out of place and unrelated to the actual audio content. Below are the details:

Model Version: Whisper v2/v3
Environment: [Operating system, Python version, etc.]
Description: The transcription unexpectedly includes the phrase "请不吝点赞 订阅 转发 打赏支持明镜与点点栏目" at the beginning.
Steps to Reproduce:
Use Whisper v2/v3 to transcribe any Chinese voice.
Additional Notes: This issue was also observed today when using ChatGPT for conversation in Chinese, but I haven't speak anything yet, just background noises caused ChatGPT to transcribe the Chinese above.

related discussion: 
https://github.com/openai/whisper/discussions/928 
https://xianote.com/why-openai-whisper-other/

Best regards,

**Comment by ojama** [comment]
Meet the same problom. I think this is caused by some junk training data LOL

**Reply by 1chunQAQ** [reply]
Is there a good solution?
